canary-checker/docs/reference/1-kubernetes-resource.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Kubernetes Resource
+sidebar_class_name: popular
+---
+
+# <Icon name="k8s" /> Kubernetes Resource
+
+The Kubernetes resource check creates kubernetes resources based on the provided manifests & perform checks on them. Some common use case of this
+check would be to see if a service is accessible via the ingress as shown in the example below.
+
+```yaml title="ingress_test.yaml"  file=../../../modules/canary-checker/fixtures/k8s/kubernetes_resource_ingress_pass.yaml
+
+```
+
+:::warning
```

Comment:
Can we create a minimal version of the of the check without a namespace specified, and then move this example + warning to an example called "Creating a new namespace for a check"